css/pages/index.css

```diff
@@ -329,17 +346,20 @@ div.handles-container {
 
 .api-event {
   margin: 15px 5px;
-  font-size: 30px;
+  font-size: 1rem;
```

Comment:
`1 rem = 16 px` and it is the default font size, no need to define it explicitly.

Reply:
done